shaka-lab-node/macos/README.md

```diff
@@ -0,0 +1,45 @@
+# Shaka Lab Node for macOS
+
+The Shaka Lab Node package provides Selenium grid nodes.
+This is the macOS package.
+
+For documentation on the package and configuration, or for links to other
+platforms, see [the general docs](../README.md#readme).
+
+**NOTE**: Browsers running in a macOS node **will** be visible.
+
+## Pre-requisites
+
+ - Get Homebrew: [https://brew.sh/#install](https://brew.sh/#install)
+
+## Installation
+
+TODO: Package distribution
+
+## Updates
+
+TODO: Package distribution
+
+## Configuration
+
+The config file is at `/opt/homebrew/etc/shaka-lab-node-config.yaml`.
+See the [configuration section](#configuration) below for details.
```

Comment:
You know, I hadn't noticed this on read-throughs of the previous PRs, but why does the configuration section contain a hyperlink that links to... itself? And why does it say that the configuration section is "below", when it's referring to itself?

Reply:
Ah, I used to have all of the platform details in one giant doc, with a common configuration section.  So this is a refactoring mistake.  The config section is now in the doc one level up.  So both the link and text are wrong.  I'll fix all three in this PR.

Reply:
Fixed.